packages/components/src/components/loaders/Progress/Progress.tsx

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 const Wrapper = styled.div`
-    background: ${({ theme }) => theme.STROKE_GREY};
+    background: ${({ theme }) => theme.borderOnElevation0};
```

Comment:
Please add ` border-radius: ${borders.radii.full};` like in [Stepper](https://github.com/trezor/trezor-suite/blob/6752b2aee2920e4f83ed066028527e46b98746c3/packages/components/src/components/loaders/Stepper/Stepper.tsx#L31) :)